ReleaseNotes/eng/Altibase_7_3_0_0_1_Release_Notes.md

```diff
@@ -0,0 +1,856 @@
+
+#### Improved Undo Tablespace Reusability
+
+To enhance the stability of Undo tablespace reuse, unnecessary associations between the undo tablespace and disk indexes have been removed, thereby eliminating potential bug-related risks. Additionally default and maximum values of related properties have been changed to improve disk page space efficiency.
+
+- INDEX_INITTRANS maximum value has been increased from 30 to 50.
+- Default and maximum values for INDEX_MAXTRANS have been changed from 30 to 50.
+
+#### 2.1.14 DBeaver Package
+
+Provide a package for DBeaver on Windows.
+
+</br>
+
+### 2.2 Changes
+
+The following describes the features that DBAs and developers need to be aware of, which include additions, modifications, and removals.
+
+#### 2.2.1 Database Version
+
+Version by Database Component
+
+| Altibase server/client Version | Database Binary Version | Meta Version | Communication Protocol Version | Replication Protocol Version |
+| :----------------------------: | :---------------------: | :----------: | :----------------------------: | :--------------------------: |
+|           7.1.0.8.8            |          6.5.1          |    8.11.1    |             7.1.7              |            7.4.7             |
+|           7.3.0.0.1            |          7.3.0          |    9.3.1     |             7.1.8              |            7.4.9             |
+
+#### 2.2.2 Compatibility
+
+##### Database Binary Version
+
+The database binary version indciates the compatibility of the database image file and log file.
+
+The database binary version has been updated due to enhancements in the logging structure of log files. As a result, Altibase 7.3 and earlier versions are not compatible, requiring migration efforts when upgrading to Altibase 7.3.
+
+##### Meta Version
+
+Given that the major version of Meta has changed, it is necessary to reconfigure the metadata when upgrading from the earlier version to Altibase 7.3.
+
+##### Communication Protocol Version
+
+The patch version of the communication protocol version has been changed. The same major and minor versions ensure client backward compatibility.
+
+> Client backward compatibility ensures that user applications developed using a older version of the Altibase library work well when connected to a newer version of Altibase.
+
+##### Replication Protocol Version
+
+Major and minor version of replication protocol version has not been changed, but the patch version has been updated. Thus, LAZY mode replication is compatible between older and newer versions, but EAGER mode replication and other additional features that require the same replication protocol version are not compatible. 
+
+> ###### Altibase Replication Backward Compatibility
+>
+> Only LAZY mode replication guarantees backward compatibility.
+>
+> While EAGER mode replication, DDL synchronization and Offline Replication do not ensure backward compatibility since they require the same replication protocol version.
+
```

Comment:
DDL synchronization does not ensure the backward compatibility because it requires all three 자리(digits?) of the replication protocol version to match.
-> DDL synchronization does not ensure the backward compatibility because it requires the identical replication protocol version. 으로 해도 되지 않을까요? 아니면 it requires that all of replication protocol versions are same.

Additional features -> Replication additional feature 가 더 나을것 같습니다.. 
 그런데,, 일단 이부분은,,

##### Replication Protocol Version
Replication protocol version indicates the compatibility of replication for both backward compatibility and compatibility with additional replication features.

Altibase replication in Lazy mode ensure the Altibase replication backward compatibility, because there is no change of major and minor version of replication protocol. However, it does not ensure the compatibility with replication additional features including Eager mode replication, because patch version has been changed.

이것만 작성하고, "Altibase Replication Backward Compatibility"는 추후에 다시 보강할께요. 국문도 좀 다듬어야 할거 같아서, 영문에서는 뺐었습니다.

Reply:
"Altibase Replication Backward Compatibility"를 제외한 Replication Protocol Version 본문을 수정하였습니다.